docs/cli_installation.md

```diff
@@ -37,6 +37,17 @@ sudo install -m 555 argocd-linux-amd64 /usr/local/bin/argocd
 rm argocd-linux-amd64
 ```
 
+#### Download latest stable version
+
+You can download the latest stable release by setting the following `VERSION`
```

Comment:
Considering users do not need to set anything to execute below commands, what do you think about rephrasing as below?
```suggestion
You can download the latest stable release by executing below steps:
```